readme.txt

```diff
@@ -135,6 +135,9 @@ Currently, the following add-ons are available for Event Tickets:
 * Fix - Correct broken javascript for themes that change the base post classes. [ET-614]
 * Tweak - Refine logic for the no results notice on the "My Tickets" page. [ETP-151]
 * Tweak - Remove duplicate ticket script to prevent conflicts. [ET-596]
+* Tweak - Change the opt-out checkbox to be checked by default. [ET-615]
```

Comment:
These changelog items need to explain which opt-out checkbox (like public Attendee List opt-out)

Reply:
OK, but it is now "one opt-out to rule them all" 💍